Source/opal/OpalSurface.m

```diff
@@ -104,7 +104,7 @@ - (void) createCGContexts
       _gsWindowDevice->gdriverProtocol |= GDriverHandlesExpose | GDriverHandlesBacking;
       _gsWindowDevice->gdriver = self;
 
-      _backingCGContext = createCGBitmapContext(
+      _backingCGContext = ctx ?: createCGBitmapContext(
```

Comment:
Shouldn't this read:

 _backingCGContext = ctx ? ctx : createCGBitmapContext(

I also think that we should rather set the _x11CGContext as this is the real context drawings end up in.